service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -2,6 +2,7 @@
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.*;
+import static org.hamcrest.Matchers.is;
```

Comment:
It looks like we already do a `.*` import above? Could we use `Matchers.is` here, rather than add the redundant import?

Reply:
Yes, in general we don't want to use wildcard (`*`) imports.

Reply:
There's an IDE setting that should prevent this from happening, I can show you how to enable it if you haven't yet.